python/ray/rllib/a3c/a3c.py

```diff
@@ -11,7 +11,7 @@
 import ray
 from ray.rllib.a3c.runner import RunnerThread, process_rollout
 from ray.rllib.a3c.envs import create_and_wrap
-from ray.rllib.common import Agent, TrainingResult, get_tensorflow_log_dir
```

Comment:
is this intended?

Reply:
Oh, seems like Eric split local_log_dir and upload_dir, that's a better way to do it than what we had :)